app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
+  <h3>Poems in this submission</h3>
+  <ul>
+    <% if @submission&.poems&.each do |poem| %>
+      <li><%=poem.title%></li>
+      <% end %>
```

Comment:
This `end` closes the `each` the next `end` closes the `if`. But the conversation around it means it's certainly not clear or idiomatic.